Kj/support page

kelseythejackson opened the pull request:
The experience/ behavior of the support section would essentially function as a "hub" that points users to the appropriate place to ask questions or get their issues solved.

Version 1: 3 Card Grid 
http://localhost:4000/docs/resources/support-1/

Version 2:  Tabs
http://localhost:4000/docs/resources/support-2/

Version 3: Split layout with checklist
http://localhost:4000/docs/resources/support-3/

kelseythejackson committed:
switching branches

kelseythejackson committed:
Merge branch 'next' into kj/support-page

kelseythejackson committed:
switching branches

kelseythejackson committed:
a few options

kelseythejackson committed:
removed empty link

claviska commented:
Nice! I'm digging the first one:

https://webawesome-git-kj-support-page-font-awesome.vercel.app/docs/resources/support-1

I have no idea how far along you are, as I stumbled on this in my email just now, but some initial thoughts:

1. Maybe change HelpScout to "Email"
2. Twitter can become something like Socials or Social Media — we have Bluesky, Twitter, Mastodon, and Threads active and running now
3. I'd love to see the Github / Community Chat / Support Topics / Socials in a more visual format like a grid or something.

---

For the second option, I'm not sure the tabs do a great job at guiding the user based on their needs.

For the third option, I like the use of cards towards the top. The buttons on the right seem redundant since the labels on the left could be clickable.

kelseythejackson committed:
tweak

kelseythejackson commented:
I updated/ simplified the first option (my favorite too) and I'm gonna work on fleshing out the socials part.

kelseythejackson committed:
switching

kelseythejackson committed:
switching

kelseythejackson committed:
Merge branch 'next' into kj/support-page

kelseythejackson committed:
another option

kelseythejackson committed:
updted page

kelseythejackson commented:
Updated the support page and the sidebar. 

**Documentation and Navigation Improvements:**

* Updated the "Help & Support" link in the sidebar (`sidebar.njk`) to point to a new internal support page instead of the GitHub discussions. This makes support resources more discoverable and keeps users within the documentation site.

**Support and Community Resources Expansion:**

* Added a comprehensive `support.md` page detailing all available support channels (GitHub issues, Discord, email) and listing social media links for staying updated. This helps users understand where and how to get help or connect with the community.

kelseythejackson committed:
updted

kelseythejackson committed:
removed purple button

claviska commented:
@kelseythejackson let's just ditch the purple. The spacing looks good then we can get this in.

kelseythejackson committed:
removed purple
